scanapi/evaluators/code_evaluator.py

```diff
@@ -19,6 +19,23 @@ class CodeEvaluator:
 
     @classmethod
     def evaluate(cls, sequence, spec_vars, is_a_test_case=False):
+        """Evaluate a sequence of python code and return a evaluation indicator.
+
+        Args:
+            sequence [string]: string representing python statements to be
+            evaluted
+            spec_vars [dict]: dictionary containing response for an API request
+            is_a_test_case[bool]: indicator for checking if the given evalution
+            is a test case
+
+        Returns:
+            (boolean, string): (True, None) for successful evalution
```

Comment:
`(boolean, string)` -> `Boolean[string]`

The docstring template : https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes

Reply:
The method returns a tuple of boolean and a string. So instead of `Boolean[string]`, I used `Tuple[Boolean, string]`

Reply:
@sijanonly the second approach looks a lot better since a `tuple` is returned. So let's do that. 😄

Reply:
You mean

> Hey @sijanonly, thank you very much for the contribution! I left some suggestions, let me know what you think.

@camilamaia Thank you for the suggestion. Those look more clear. I will update.